Sources/Client/Utils/AssociatedValue.swift

```diff
@@ -10,10 +10,10 @@ import Foundation
 
 // MARK: Vars for Extensions
 
-func associated<T>(to base: AnyObject,
-                   key: UnsafePointer<UInt8>,
-                   policy: objc_AssociationPolicy = .OBJC_ASSOCIATION_RETAIN,
-                   initialiser: () -> T) -> T {
+public func associated<T>(to base: AnyObject,
```

Comment:
Why do we need these public?

Reply:
Because now it uses in LLC and Core 🤷‍♂️